Packed Update

Peptide90 opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Nothing drastic just a collection of fixes that were requested awhile back and some up to date content changes:

- Added a table and windoor between hydroponics and kitchen as well as between kitchen and bar so chef can serve people directly.
- Added windoor to bar directly so they no longer have to climb over or go through kitchen.
- Added 4x drones to engineering.
- Extended LV cables at arrivals so they fully reach the exterior doors.
- Changed airlocks at arrivals so they're all now docks.
- Removed that obnoctious wall thing at arrivals that stopped shuttles getting in close to dock.
- Generally redesigned window layout at arrivals (see screenshot) to reduce the likelihood of it being spaced. Also added caution decals to the plating in the airlocks.
- Fixed the SMES wiring so that the substation in engineering is now on the distribtion side rather than the generation side.
- Added a new substation in atmos to reduce the load on the engineering substation.
- Added spawn for QM.
- Added spawns for salvage and changed the airlocks in the mining room to all be salvage locked now.
- Medbay now has shutters to stop people climbing over tables.
- Added box of bodybags to morgue.
- Added correct atmos fix markers to the nitrogen and oxygen atmos tanks so they roundstart with gas rather than vacuum now. Fixes #5291

**Screenshots**
<!-- If applicable, add screenshots to showcase your PR. If your PR is a visual change, add
screenshots or it's liable to be closed by maintainers. -->
![packed1](https://user-images.githubusercontent.com/78795277/154166491-17f751f9-4593-4d96-980f-3b2499acc6cb.PNG)

![packedbar](https://user-images.githubusercontent.com/78795277/154166507-33a4ae10-fc47-4985-8089-5cf11b228283.PNG)


**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl:
- tweak: Packed has a bunch of small updates! Bar and kitchen have windoors now so they have better access.
- tweak: Medbay has shutters to stop the greytide. Engineering now have drones. Salvage have their own room and spawns now.
- tweak: Arrivals has been tweaked to have shuttle docks! And more resistance to being spaced.

mirrorcult commented:
need @TimrodDX approval

TimrodDX commented:
Here is my review. The only changes I do not approve of are the ones to arrivals and some of the ones to the kitchen and bar.

The arrivals change I disapprove of because the whole "grass inside glass" thing is a hallmark of the post-Metastation mapping era, and Packed is from well before that was ever a thing. I get that arrivals being spaced is an issue, but that's an issue that will largely be resolved once the arrivals shuttle is in place.

The bar changes I also disapprove of because that's not how they are on the original map. The way the kitchen is laid out was a quick-fix to a couple of early issues that arose because there was no windoor keyed to both kitchen and bar access. What you could do (and what I would gladly accept) is adding that windoor and changing the area to look like the original map:

![image](https://user-images.githubusercontent.com/7907891/153782421-a8b4ffac-d353-4031-82a2-14491b9e7c96.png)

The table between the kitchen and botany can stay though. That's perfectly fine.

Peptide90 commented:
> Here is my review. The only changes I do not approve of are the ones to arrivals and some of the ones to the kitchen and bar.
> 
> The arrivals change I disapprove of because the whole "grass inside glass" thing is a hallmark of the post-Metastation mapping era, and Packed is from well before that was ever a thing. I get that arrivals being spaced is an issue, but that's an issue that will largely be resolved once the arrivals shuttle is in place.
> 
> The bar changes I also disapprove of because that's not how they are on the original map. The way the kitchen is laid out was a quick-fix to a couple of early issues that arose because there was no windoor keyed to both kitchen and bar access. What you could do (and what I would gladly accept) is adding that windoor and changing the area to look like the original map:
> 
> ![image](https://user-images.githubusercontent.com/7907891/153782421-a8b4ffac-d353-4031-82a2-14491b9e7c96.png)
> 
> The table between the kitchen and botany can stay though. That's perfectly fine.

Largely reverted arrivals but had no image to work from. Converted it to more reinforced walls and a double window in the center. Let me know if it's still too much and I'll load up a clean copy to copy off.

Also removed the changes to the kitchen area and followed your screenshot. Added the kitchen/hydro and kitchen/bar windoor types so that it's all workable. Hopefully that table for kitchen doesn't look too ass without the curtain.

In addition to these changes since my glass doors PR got merged I changed the HoP, CMO and RD doors to glass variants since the whole front of their offices is glass anyway and changed most of the external doors where there's a dock, including the docks to glass variants so people have some chance of docking etc. See screenshots above.
